ideas42ventures/ideas42ventures.com
# Added content for VSP pages + created custom components

Works on #25 

- Added the content for the VSP pages

- Created custom components for: the sub banner, join this cohort call to action, also when you're ready to apply section.

**How to test:** 

1. The links below should lead to valid URLs with their respective content on each page.

- [ventures-for-shared-prosperity](https://deploy-preview-26--ideas42ventures.netlify.app/ventures-for-shared-prosperity/)

- [focus-area](https://deploy-preview-26--ideas42ventures.netlify.app/ventures-for-shared-prosperity/focus-area)

- [what-we-provide](https://deploy-preview-26--ideas42ventures.netlify.app/ventures-for-shared-prosperity/what-we-provide/)

- [apply](https://deploy-preview-26--ideas42ventures.netlify.app/ventures-for-shared-prosperity/apply)
